src/screens/DiscoverScreen.tsx

```diff
@@ -1,30 +1,233 @@
 /**
- * Requests trending movies, passing data to ShowCard components.
+ * Requests a variety of filtered shows, rendering carousels
  * @returns {JSX.Element}
  */
 export default function DiscoverScreen(): JSX.Element {
-    const { profile, setProfile } = useProfileContext();
     const { trendingShows, loading } = useTrendingShows('alpha');
+    const [highestRated, setHighestRated] = useState<ShowData[] | null>(null);
+    const [newlyAdded, setNewlyAdded] = useState<ShowData[] | null>(null);
+    const [actionAdventure, setActionAdventure] = useState<ShowData[] | null>(null);
+    const [comedy, setComedy] = useState<ShowData[] | null>(null);
+    const [horror, setHorror] = useState<ShowData[] | null>(null);
+    const [popularNetflix, setPopularNetflix] = useState<ShowData[] | null>(null);
+    const [popularPrime, setPopularPrime] = useState<ShowData[] | null>(null);
+    const [popularHulu, setPopularHulu] = useState<ShowData[] | null>(null);
+    // Loading states
+    const [highestRatedLoading, setHighestRatedLoading] = useState<boolean>(true);
+    const [newlyAddedLoading, setNewlyAddedLoading] = useState<boolean>(true);
+    const [actionAdventureLoading, setActionAdventureLoading] = useState<boolean>(true);
+    const [comedyLoading, setComedyLoading] = useState<boolean>(true);
+    const [horrorLoading, setHorrorLoading] = useState<boolean>(true);
+    const [popularNetflixLoading, setPopularNetflixLoading] = useState<boolean>(true);
+    const [popularPrimeLoading, setPopularPrimeLoading] = useState<boolean>(true);
+    const [popularHuluLoading, setPopularHuluLoading] = useState<boolean>(true);
 
-    // TODO: #194 Make skeleton loading screen
-    if (loading) return <p>Loading...</p>;
+    const discoverHandler = async (
+        movieParams: DiscoverMovie,
+        tvParams: DiscoverTv,
+        setState: React.Dispatch<React.SetStateAction<ShowData[] | null>>,
+        setLoading: React.Dispatch<React.SetStateAction<boolean>>
+    ) => {
+        const movies: ShowData[] | null = await getDiscoverMovies(movieParams);
+        const tv: ShowData[] | null = await getDiscoverTv(tvParams);
+        const shows: ShowData[] | null = [];
+
+        if (movies && tv) shows.push(...movies, ...tv);
+        setState(shows);
+        setLoading(false);
+    };
+
+    useEffect(() => {
+        const highRatedHandler = async () => {
+            const params: DiscoverMovie | DiscoverTv = {
+                pages: 1,
+                sort_by: 'popularity.desc',
+                vote_average_gte: 8.0,
+                vote_count_gte: 2000,
+            };
+            discoverHandler(params, params, setHighestRated, setHighestRatedLoading);
+        };
+
+        const newlyAddedHandler = async () => {
+            const params: DiscoverMovie | DiscoverTv = {
+                pages: 1,
+                sort_by: 'popularity.desc',
+                vote_count_gte: 2000,
+                // TODO: #613
```

Comment:
Make sure to run `npm run todo-ci` and maybe add a short description after the number here.